core/models/src/node/tx.rs

```diff
@@ -43,13 +43,13 @@ impl AsRef<[u8]> for TxHash {
 }
 
 impl TxHash {
-    pub fn to_hex(&self) -> String {
-        format!("0x{}", hex::encode(&self.data))
+    pub fn to_str(&self) -> String {
```

Comment:
I think it is slightly better to call it `to_string`, because `str` usually implies `&str` type